src/Servers/IIS/Directory.Build.props

```diff
@@ -9,20 +9,20 @@
     <HostArch Condition="'$(Platform)' == 'Win32'">x86</HostArch>
     <HostArch Condition="'$(Platform)' == 'AnyCPU'">x64</HostArch>
     <HostArch Condition="'$(DotNetBuild)' == 'true'">$(TargetArchitecture)</HostArch>
-    <LibNetHostPath>$(NuGetPackageRoot)microsoft.netcore.app.host.win-$(HostArch)\$(MicrosoftNETCoreAppRefVersion)\runtimes\win-$(HostArch)\native</LibNetHostPath>
+    <LibNetHostPath>$(NuGetPackageRoot)microsoft.netcore.app.host.win-$(HostArch)\$(LibNetHostAppPackVersion)\runtimes\win-$(HostArch)\native</LibNetHostPath>
     <!-- This tools version MUST match the same version as runtime uses to compile libnethost.lib -->
     <PlatformToolsetVersion>143</PlatformToolsetVersion>
   </PropertyGroup>
 
   <!-- Platform is different during restore than during build. Therefore, restore everything when not building the product. -->
```

Comment:
maybe we can flow this in via GenerateFiles.csproj too?

Reply:
You mean the platform that should be used at restore time?

Reply:
yes or restoring the packages in general

Reply:
I _think_ the problem that required this is that the platform negotiation happens at P2P build time. NuGet doesn't see the agreed upon Platform as it trims out P2P properties. The outer context, i.e. GenerateFiles.csproj or Tools.props wouldn't have more information than the infrastructure in this file.